src/actions/userActions.ts

```diff
@@ -52,6 +54,23 @@ export const setRegistrationToken = (
     },
 })
 
+export const sendRegistrationToken = (
+    t: any,
+    registrationToken: string
+) => async (dispatch, getState) => {
+    const { user: userState } = getState()
+    const { token, userId } = userState.user
+
+    try {
+        const api = new Api(token)
+        await api.addRegistrationToken(userId, registrationToken)
+        dispatch(setRegistrationToken(registrationToken))
+    } catch (e) {
+        console.log('addRegistrationToken error: ', e)
+        dispatch(showErrorMessage(t('errors.register')))
```

Comment:
we could have a generic error in french + a detail in english, so we doesn't need to manage it entirely.

Reply:
For the moment I've put both translations I think, but I'm not against your idea